tls/test_tls_handshake.py

```diff
@@ -465,3 +465,67 @@ def test(self):
         self.assertTrue(res, "Bad second handshake: %s" % res)
         res = x509_check_issuer(vhs.cert, "New Issuer")
         self.assertTrue(res, "Wrong certificate reloaded")
+
+class CloseConnection(tester.TempestaTest):
```

Comment:
Test documentations are missing, need to describe purpose of each test. Helps to observe coverage without reading the full test code.